Crash consistent VM restore points (preview)/createRestorePointCollection.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "VMName": {
+      "type": "string",
+      "defaultValue": "<VMName>"
+    },
+    "VMId": {
+      "defaultValue": "subscriptions/<SubscriptionID>/resourceGroups/<ResourceGroupName>/providers/Microsoft.Compute/virtualMachines/<VMName>",
+      "type": "String"
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Compute/restorePointCollections",
+      "name": "[concat(parameters('VMName'),'_rpc')]",
```

Comment:
For this also, we will put the name completely customizable?

Reply:
This also like this: "[parameters('RestorePointCollectionName')]"

Reply:
Done. @prchin please check once again if its correct